### ISSUE TITLE
Drag and drop from master palette not working

### ISSUE THREAD

**Issue by Ombrezz** [musescore/MuseScore#17411]
Title says it all, I can't drag and drop anything from the master palette onto the score. I'm on Fedora Linux 38, this happens on Musescore 4 stable and nightly appimages as well as the rpm package which is on Musescore 3. Having something on the score selected and double clicking the element on the master palette still works though. Here's a video demonstrating the issue.

[Screencast from 2023-04-29 13-29-50.webm](https://user-images.githubusercontent.com/37753837/235317283-cca7977b-8f05-4e84-a824-beae944fa6be.webm)

**Comment by cbjeukendrup** [comment]
That's not the master palette but the normal palette... (the master palette is the dialog that you get when you go to View > Master Palette…).

It might be related to #11580, although that issue _is_ about the master palette, and there the normal palette does work.

Since for drag&drop we depend on Qt, which in turn depends on the OS, I'm afraid we can't do much about this, especially since it does work on other operating systems. I've searched a bit on the internet and found something that seems related: https://gitlab.gnome.org/GNOME/mutter/-/issues/2715. It looks like a patch was merged to that repository 1 day ago. That _might_ be the fix for this issue, but I don't know when you'll receive it (as an update for your OS).
